### PR TITLE
[MME][DOCKER] Adding support for RHEL8

### PR DESCRIPTION
**Pull request by rdefosse**
## Summary

Based on Lionel Gauthier work on `centos8` I retried it (I failed back in November) and it works fine now.

For us it is very important, since we will be able to generate proper images for our OpenShift cluster.

## Test Plan

As described in the `lte/gateway/docker/mme/README.md` file, I was able to build a target image for RHEL8.

Next steps:

- will be to run and see if it does connect properly with an HSS instance.
- add RHEL8 build stage to the `OAI-DsTester` pipeline

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Comment by rdefosse** [comment]
MAGMA-OAI-TESTS

**Comment by rdefosse** [comment]
MAGMA-OAI-TESTS

File: lte/gateway/docker/mme/Dockerfile.rhel8 [hunk 1]
```diff
@@ -0,0 +1,545 @@
...
+      net-tools \
+      tzdata && \
+    echo "/usr/local/lib" > /etc/ld.so.conf.d/local-lib.conf && \
+    echo "/usr/local/lib64" >> /etc/ld.so.conf.d/local-lib.conf && \
+    yum clean all -y && \
+    rm -rf /var/cache/yum
+
+# Copy runtime-used shared libraries from builder
+WORKDIR /lib64
+COPY --from=magma-mme-builder /lib64/libsctp.so.1 .
```
Comment (mattymo) [inline lte/gateway/docker/mme/Dockerfile.rhel8:473]: This is the longest dockerfile I've ever seen. This sort of stuff should be handled by packages, not explicit dockerfile copies

**Review by 119Vik — APPROVED** [review]

**Review by mattymo — APPROVED** [review]

File: lte/gateway/docker/mme/Dockerfile.rhel8 (hunk 1 above)
Comment (rdefosse) [reply lte/gateway/docker/mme/Dockerfile.rhel8:473]: sorry.

When I have time, I will re-factor (it is the same on the Ubuntu 18 version)

**Review by lionelgo — APPROVED** [review]

**Review by tmdzk — APPROVED** [review]

**Pull request merged**